config/routes.js

```diff
@@ -0,0 +1,10 @@
+
+const MarkerAPIController = require("../src/Controllers/MarkerAPIController");
+const AddNewRouteController = require("../src/Controllers/AddNewRouteController");
+
+function routes(app){
```

Comment:
Don't forget you need to document your API! Suggest adding something into the readme for each endpoint (this is a good format https://gist.github.com/iros/3426278).

Remember this is as important as the API itself